src/rvsdg/live_variables.rs

```diff
@@ -173,6 +173,7 @@ impl VarSet {
         self.vars.count_ones(..)
     }
 
+    #[allow(dead_code)]
```

Comment:
If these are dead, can we remove them? (if they're just used in tests, let's annotate them as `#[cfg(test)]`

Reply:
yep, done